### PR TITLE
Porting core p7 2

### PR DESCRIPTION
**Pull request by jkeenan**
@atoomic, since these commits entailed running various `regen` scripts, they will need more careful review.

### CODE REVIEW

File: t/porting/copyright.t [hunk 1]
```diff
@@ -27,6 +27,7 @@ if ( $Config{usecrosscompile} ) {
 }
 
 my ($opt) = @ARGV;
+$opt ||= '';
```
Comment (atoomic) [inline t/porting/copyright.t]: this could probably use safely `//=` instead as this is a porting tool?

File: regen/embed.pl [hunk 2]
```diff
@@ -46,7 +46,7 @@ ($)
     $error_count++;
 }
 
-sub full_name ($$) { # Returns the function name with potentially the
+sub full_name { # Returns the function name with potentially the
```
Comment (atoomic) [inline regen/embed.pl]: alternatively we could consider using `sub full_name :prototype($$)`

File: t/porting/copyright.t (hunk 1 above)
Comment (jkeenan) [reply t/porting/copyright.t]: Okay, see 2021c39cbb5f598a82bcd552898cb09c3a0690f3

File: regen/embed.pl (hunk 2 above)
Comment (jkeenan) [reply regen/embed.pl]: Since I've never used prototypes at all, I'll have to trust you on that!  See 7aad926102

**Comment by jkeenan** [comment]
@atoomic This is where I'll leave off for today.  Please review and if okay, merge into core-p7.  Thx.

**Review by atoomic — APPROVED** [review]

**Pull request merged**